### PR TITLE
Add score module SDK and update demo-score

### PR DESCRIPTION
**Pull request by belyrs**

### CODE REVIEW

File: packages/modules/src/scoring/Scoring.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  const hexDigest = Crypto.u8aToHex(digest)
+  return hexDigest
+}
+
+export function getUriForScore(journalContent: IJournalContent) {
+  const scoreDigest = generateDigestFromJournalContent(journalContent)
+  return Identifier.hashToUri(scoreDigest, SCORE_IDENT, SCORE_PREFIX)
+}
+
+export async function fetchAverageScore(
```
Comment (smohan-dw) [inline packages/modules/src/scoring/Scoring.ts]: Rename this as fetchScore. Also, move this function to Score.chain. We should all non-chain functions within Score.ts and and the ones that require an interaction with the chain in Score.chain.ts.

File: packages/modules/src/scoring/Scoring.chain.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import {
+    ScoreType,
+    IScoreDetails,
+    SCORE_MODULUS,
+    IJournalContent,
+  } from '@cord.network/types'
+  import { ConfigService } from '@cord.network/config'
+  import { Identifier, SDKErrors } from '@cord.network/utils'
+  import * as Did from '@cord.network/did'
+
```
Comment (smohan-dw) [inline packages/modules/src/scoring/Scoring.chain.ts:10]: Missing toChain function.

File: packages/modules/src/scoring/Scoring.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+export function ComputeAverageRating(rating: number, count: number): number {
+  return rating / count
+}
+
+export function generateDigestFromJournalContent(
+  journalContent: IJournalContent
+) {
+  const digest = Crypto.hash(JSON.stringify(journalContent))
+  const hexDigest = Crypto.u8aToHex(digest)
+  return hexDigest
```
Comment (smohan-dw) [inline packages/modules/src/scoring/Scoring.ts:28]: We need a function, that will take a score input and produce the transformed content that can be used with the toChain function to anchor the details to the chain. The return value should have all the elements required for the extrinsic

File: packages/did/src/DidDetails/FullDidDetails.ts [hunk 4]
```diff
@@ -34,6 +34,7 @@ const methodMapping: Record<string, VerificationKeyRelationship | undefined> = {
   'did.submitDidCall': undefined,
   didLookup: 'authentication',
   didName: 'authentication',
+  score:'authentication',
```
Comment (smohan-dw) [inline packages/did/src/DidDetails/FullDidDetails.ts]: Should be assertionMethod here. Authentication should only be used for DID management-related operations.

File: packages/modules/src/index.ts [hunk 5]
```diff
@@ -4,6 +4,7 @@ export * as Registry from './registry/index.js'
 export * as Content from './content/index.js'
 export * as Stream from './stream/index.js'
 export * as Document from './document/index.js'
+export * as Scoring from './scoring/index.js'
```
Comment (smohan-dw) [inline packages/modules/src/index.ts]: Rename this to Score

File: demo/src/demo-score.ts [hunk 6]
```diff
@@ -1,93 +1,160 @@
...
   )
-  console.dir(chainScore, { depth: null, colors: true })
+  console.log(`\n✅ Registry Authorization - ${registryAuthority} - created!`)
 
-  const chainAvgScore = await Cord.Score.queryAverage(
-    journalContent.entity,
-    journalContent.scoreType
+  console.log(`\n❄️  Journal Entry `)
+  let journalContent: IJournalContent = {
+    entity: sellerDid.uri.replace('did:cord:', ''),
```
Comment (smohan-dw) [inline demo/src/demo-score.ts]: entity Identifier is not DID atm. It's an identifier generated from a Govt ID without any keys attached to it. Please refer the specs.

File: packages/modules/src/scoring/Scoring.ts (hunk 3 above)
Comment (belyrs) [reply packages/modules/src/scoring/Scoring.ts:28]: Have made a separate PR for this issue. Please review: https://github.com/dhiway/cord.js/pull/135

**Pull request closed without merging**